### DISCUSSION TITLE
Firefox version?

### DISCUSSION THREAD

**Discussion by Spectre-856** [tsukasa/BdBrowser#34] [Ideas]
Idk if is already available but if not... Is there a way to make this project to Firefox users? maybe a raw code? Idk how much difficult is, but, if possible, here's the idea.

**Comment by tsukasa** [comment]
Greetings,

I explored that option in 2023 and concluded that this endeavour would prove challenging for me. One of the first things I did to Strencher's extension was to convert it to Manifest V3 because Google originally announced that starting January 1st 2023, all Manifest V2 extensions would cease to function.

Although Manifest V2 extensions are still alive a good year later, the implication is that Firefox would have to support certain V3 specs and features, like the service workers BdBrowser is using. To the best of my knowledge, this is currently not (or only partially) the case.

So for the foreseeable future, BdBrowser will only be compatible with Chrome (and derivatives).
